packages/cli/src/module-resolver.ts

```diff
@@ -339,5 +358,10 @@ export function getCommonJsExports(
   }
 
   const { exports, reexports } = parse(code);
-  return [...exports, ...reexports];
+
+  // Re-exports are paths to exports that must be resolved themselves
+  const resolvedReexports = reexports.flatMap((reexport: string) =>
+    getCommonJsExports(reexport, system, path),
+  );
+  return [...new Set([...exports, ...resolvedReexports])];
```

Comment:
Is it necessary to turn this back into an array? Can we just return a `Set`?

Reply:
Definitely, I was just keeping the existing API, but we can change the return type

Reply:
Fine to change it since it's an internal API.